angular2-jwt.ts

```diff
@@ -56,57 +64,74 @@ export class AuthHttp {
 
   private _config: IAuthConfig;
   public tokenStream: Observable<string>;
+  private jwtHelper:JwtHelper;
 
   constructor(options: AuthConfig, private http: Http) {
     this._config = options.getConfig();
+    this.jwtHelper = new JwtHelper();
 
     this.tokenStream = new Observable((obs: any) => {
       obs.next(this._config.tokenGetter())
     });
   }
 
-  _request(url: string | Request, options?: RequestOptionsArgs) : Observable<Response> {
+  _request(req: Request) : Observable<Response> {
```

Comment:
No, it's not unused. It was meant to be request, but when I originally wrote it, there was already a request that I didn't want to break the API on. I was going to make a pr to make it public just as soon as the latest changes to HTTP were released in the angular 2 repository.